sdks/python/src/opik/evaluation/models/litellm_chat_model.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Any, Dict, List
+
+import litellm
+from litellm.types.utils import ModelResponse
+
+from . import base_model
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LiteLLLMChatModel(base_model.OpikBaseModel):
+    def __init__(
+        self,
+        model_name: str = "gpt-3.5-turbo",
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(model_name=model_name)
+
+        self._model_kwargs: Dict[str, Any] = kwargs
+        self._model_kwargs = self.cleanup_params(self._model_kwargs)
+
+        self._engine = litellm
+
+    def cleanup_params(self, params: Dict[str, Any]) -> Dict[str, Any]:
+        supported_params = set(
+            litellm.get_supported_openai_params(
+                model=self.model_name,
+            )
+        )
+
+        result = {}
+
+        for key in params:
+            if key not in supported_params:
+                LOGGER.warning(f"Unsupported parameter: '{key}' will be ignored.")
+            else:
+                result[key] = params[key]
+
+        return result
+
+    def generate(self, input: List, **kwargs: Any) -> str:
+        response = self.generate_ext(input=input, **kwargs)
+        return response.choices[0].message.content
+
+    def generate_ext(self, input: List, **kwargs: Any) -> ModelResponse:
```

Comment:
1. I like the idea of having a separate function that repeats provider call :+1:
2. Maybe for consistency it should have `*args` and `**kwargs` signature, without the `input` argument. The point is to have a proxy for provider's API with the same arguments
3. Adding a docstring with the link to litellm's `completion` function documentaion would be nice here.
4. Probably `provider_generate`, `engine_generate` or something like that would be a better choice here for readability.
5. In the future, after we migrate to litellm model instead of a langchain one, it will probably make sense to move this method to a base class.
6. Everything described above is applicable for async version of the method as well :)